src/common/home/Home.jsx

```diff
@@ -30,6 +34,24 @@ const Home = () => {
     });
   }, [data, setSearchTerm, searchTerm, setFilterQuery]);
 
+  //* array of tweet IDs to show on the home page
+    const tweetIdArray = [
```

Comment:
@Anirban-1490 as you mentioned
> This was done by getting 10 tweets IDs and generate as a static result rather than dynamic as twitter API dosen't shows results for more than 7 days

What is the proposal to make the cards dynamic? It won't be easy to maintain these tweet IDs manually over the lifetime of the application

Reply:
@koustov It's a conscious decision to get static mentions and change them manually every month. The Twitter API for developers doesn't allow you to pull more than 100 tweets on a call. These 100 tweets include the replies to the main mentioned tweet. So, you may end up just fetching 1 or 2 in some cases.

If we have a special Twitter account(like a scholar, scientist, etc.) then the limit is much higher. Hence most open-source app follows this paradigm. The discussion is in our Discord channel under ideas.